swagger-spec/logs/detail.yaml

```diff
@@ -1 +1,67 @@
 # /logs/{log_id}/
+get:
+  summary: Retrieve a Log
+  description: >-
+    Details about a given Log. *Read-only*.
+
+    On the front end, logs show record and show actions done on the OSF. 
+
+    ##Log Attributes
+
+    OSF Log entities have the "logs" `type`.
+
+
+    name           type                   description
+
+    ----------------------------------------------------------------------------
+
+    date           iso8601 timestamp      timestamp of Log creation
+    action         string                 Log action (see list above)
+
+
+    ##Relationships
+
+
+    ###Node
+
+
+    The node this log is associated with.
+
+
+    ###Original Node
+
+
+    The original node that was associated with this log, in case this log was a remnant of when the original node was forked or registered.
+
+
+    ###User
+
+
+    The user who performed the logged action.
+
+
+    ##Links
+
+
+    See the [JSON-API spec regarding
+    pagination](http://jsonapi.org/format/1.0/#fetching-pagination).
+
+
+    ##Query Params
+
+    Logs may be filtered by their `action` and `date`.
+
+  parameters:
+    - description: ''
```

Comment:
Can you add a simple description ([like this](https://github.com/caseyrollins/swagger-ui/blob/master/swagger-spec/preprints/detail.yaml#L16)) here?